realm/realm-library/build.gradle

```diff
@@ -128,9 +128,9 @@ task javadoc(type: Javadoc) {
         locale = 'en_US'
         overview = 'src/overview.html'
 
-        links "http://docs.oracle.com/javase/7/docs/api/"
+        links "https://docs.oracle.com/javase/7/docs/api/"
         links "http://reactivex.io/RxJava/javadoc/"
```

Comment:
rxJava is using github's certification and seems we cannot solve the browser warning :(